test/ussd_clinic.test.js

```diff
@@ -243,6 +308,22 @@ describe("app", function() {
 
                 it("entering once, timing out, redialing (session:close detected)", function() {
                     return tester
+                        .setup(function(api) {
+                            api.http.fixtures.add(fixtures_IdentityStoreDynamic.identity_search({
+                                'msisdn': '+27820001001'
+                            }));
+                            api.http.fixtures.add(fixtures_StageBasedMessagingDynamic.active_subscriptions());
+                            api.http.fixtures.add(fixtures_JembiDynamic.momconnect_exists("123456", "test clinic"));
+                            api.http.fixtures.add(fixtures_MessageSenderDynamic.send_message({
+                                to_identity: "cb245673-aa41-4302-ac47-00000001002",
+                                content: "Please dial back in to *120*550*2# to complete the pregnancy registration."
+                            }));
+                            api.http.fixtures.add(fixtures_Pilot().exists({
+                                number: '+27123456789',
```

Comment:
What number is this? `+27123456789`

Reply:
This is the number of our whatsapp account (the number that the user sees the message come from).

It's set in the application config at the top of the file in the test setup.

Reply:
Is this the same for the CHW line?

Reply:
Yes, this will be the same for all of the MomConnect USSD lines, since we use the same WhatsApp line for all MomConnect messages.

Reply:
I see, thanks Rudi